opendevin/runtime/plugins/agent_skills/agentskills.py

```diff
@@ -34,6 +35,12 @@
 ENABLE_AUTO_LINT = os.getenv('ENABLE_AUTO_LINT', 'false').lower() == 'true'
 
 # OPENAI
+"""
+https://github.com/OpenDevin/OpenDevin/pull/2052
+NOTE: GPTSwarm need manually export...
+TODO: Fix it
+export SANDBOX_ENV_OPENAI_API_KEY="sk-***"
+"""
 OPENAI_API_KEY = os.getenv(
     'OPENAI_API_KEY', os.getenv('SANDBOX_ENV_OPENAI_API_KEY', '')
```

Comment:
Within the sandbox, this would be exported again without the prefix, i.e. as "OPENAI_API_KEY".